source/common/config/resources.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <string>
+
+#include "common/common/singleton.h"
+
+namespace Envoy {
+namespace Config {
+
+/**
+ * Constant Type URLs.
+ */
+class TypeUrlValues {
```

Comment:
Should this go in the "well known" file? Don't care just a thought.

Reply:
I prefer to draw a line between the actual config and the delivery vehicle constants, so `resources.h` seems a better choice I think.